docs/project/about.md

```diff
@@ -33,20 +33,20 @@ community-driven open-source project. The decision making process is outlined in
 ## Citing
 
 If you use Pyodide for a scientific publication, we would appreciate citations.
-Please find us [on Zenodo](https://zenodo.org/record/5135072) and use the citation
+Please find us [on Zenodo](https://zenodo.org/record/5156931) and use the citation
 for the version you are using. You can replace the full author
 list from there with "The Pyodide development team" like in the example below:
 
 ```
-@software{michael_droettboom_2021_5135072,
+@software{pyodide_2021,
   author       = {The Pyodide development team},
   title        = {pyodide/pyodide},
-  month        = jul,
+  month        = aug,
   year         = 2021,
   publisher    = {Zenodo},
-  version      = {0.18.0a1},
-  doi          = {10.5281/zenodo.5135072},
-  url          = {https://doi.org/10.5281/zenodo.5135072}
+  version      = {0.18.0},
+  doi          = {10.5281/zenodo.5156931},
+  url          = {https://doi.org/10.5281/zenodo.5156931}
```

Comment:
I don't think we would necessarily need to update this for each release, but just so it doesn't point to an alpha release.